### ISSUE TITLE
Docs: Custom log formatter example does not show source for `MyCompanyLogFormatter.ts`

### ISSUE THREAD

**Issue by cromo** [aws-powertools/powertools-lambda-typescript#1256]
### What were you searching in the docs?

I was reading the docs from top to bottom and came across this since some might want to write string logs instead of using structured logging.

### Is this related to an existing documentation section?

https://awslabs.github.io/aws-lambda-powertools-typescript/latest/core/logger/#custom-log-formatter-bring-your-own-formatter

### How can we improve?

```Markdown
It looks like the source for `handler.ts` was mistakenly copied into the block below it for `utils/formatters/MyCompanyLogFormatter.ts` instead of showing an implementation of the `LogFormatterInterface`. That block should show how to implement this.
```

### Got a suggestion in mind?

Change the `utils/formatters/MyCompanyLogFormatter.ts` code block to show an implementation of the `LogFormatterInterface` as expected; as written the documentation does little more than tell you that custom formatters can be used but not how to make them.

### Acknowledgment

- [X] I understand the final update might be different from my proposed suggestion, or refused.

**Comment by dreamorosi** [comment]
Hello, thanks for reporting this.

Someone reported this yesterday #1253 and a fix was merged with #1254.

The updated/fixed version is already available on the `/dev` version of the docs.

The updated docs will be published on `/latest` tomorrow.

I'll leave the issue open until then so that others can see it if needed.

**Comment by dreamorosi** [comment]
The updated docs are now live, you can see the code [example here](https://awslabs.github.io/aws-lambda-powertools-typescript/latest/core/logger/#custom-log-formatter-bring-your-own-formatter).

Note that if you don't see the change yet it's probably due to caching, you might need to refresh your browser's cache or try with a different browser/private session to see it immediately - but it's there!
